src/stores/layers.ts

```diff
@@ -103,12 +106,29 @@ export const useLayersStore = defineStore('layers', () => {
     visibilities.value._traveltimeArrow = newVisibilities._traveltimeArrow
   }
 
+  const checkedMapButton: Ref<boolean> = ref(false)
```

Comment:
You don't need to create this variable, since the value can be deduced from the visibility of `alternativeRennesOrtho`. Maybe create a function to get the value from the `alternativeRennesOrtho` visibility. This way, you don't need to manage its value like in the lines 114 and 120.